src/actions.js

```diff
@@ -44,3 +46,6 @@ export const initializeCurrentLocation = (location: Location) => ({
   type: LOCATION_CHANGED,
   payload: location
 });
+
+export const block = (callback: Function) => ({ type: BLOCK, callback });
```

Comment:
@todesignandconquer per the docs (https://github.com/reacttraining/history#blocking-transitions) the signature of the function looks like this:

```Javascript
(location, action) => { ... }
```

I would assume at a minimum we could attribute it thusly:

```Javascript
type Location = { ... }
type Action = { ... }
... (callback(Location, Action)) => ...
```

The location type is _basically_ this: https://github.com/lanetix/ui/blob/master/client/apps/company-settings/types.js#L31-L37 and I'm not sure what the action type is.

And, to be clear, this is super non-blocking. I'm just in favor of tighter types if we have the ability to provide them.